bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
+CGCON-BCPM01-15009,R00027,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,9,,,,TO BE DELETED,
+CGCON-BCPM01-17009,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2017-009,,,,TO BE DELETED,
+CGCON-BCPM01-50001,OL000278 R00049,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0001,,,,CP04OSSM-00004-CPM1,Woods Hole Oceanographic Institution
+CGCON-BCPM01-50002,OL000097,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0002,,,,GA01SUMO-00002-CPM1,Woods Hole Oceanographic Institution
+CGCON-BCPM01-50003,OL000075,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0003,,,,,Woods Hole Oceanographic Institution
+CGCON-BCPM01-50004,OL000260,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0004,,,,CP03ISSM-00004-CPM1,Woods Hole Oceanographic Institution
+CGCON-BCPM01-50005,OL000228,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0005,,,,CP01CNSM-00005-CPM11,Woods Hole Oceanographic Institution
+CGCON-BCPM01-50006,OL000523,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0006,,,,CP01CNSM-00004-CPM1,Woods Hole Oceanographic Institution
+CGCON-BCPM01-50007,OL000323,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0007,,,,CE02SHSM-00003-CPM11,Oregon State University
+CGCON-BCPM01-50008,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0008,,,,CE09OSSM-00005-SBC11,Oregon State University
+CGCON-BCPM01-50009,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2015-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BCPM01-60002,OL000593,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2016-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BCPM01-60701,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,60701,,,,DELETE?,
+CGCON-BCPM01-70001,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2017-0001,,,,CP04OSSM-00006-CPM1,Woods Hole Oceanographic Institution
+CGCON-BCPM01-70009,CGCON-BCPM01-17009,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2017-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BCPM01-70009,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,UNKNOWN,,,,,Woods Hole Oceanographic Institution
```

Comment:
I believe the ASSET_UID for this one is supposed to be CGCON-BCPM01-99999 (according to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet.